docs/excel/custom-functions-json-autogeneration.md

```diff
@@ -1,21 +1,77 @@
 
 [!include[Excel custom functions note](../includes/excel-custom-functions-note.md)]
 
+## CustomFunctionsMetadataPlugin
+
+The plugin is [CustomFunctionsMetadataPlugin](https://github.com/OfficeDev/Office-Addin-Scripts/blob/master/packages/custom-functions-metadata-plugin/README.md). To install and configure it, use the following steps.
+
+> [!NOTE]
+> 
+> - The tool can be used only in a NoteJS-based project.
+> - These instructions assume that your project uses [Webpack](https://webpack.js.org/) and that you have it installed and configured.
+> - If your custom function add-in project is created with the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md), Webpack is installed and all of these steps are done automatically, but when applicable, you must do the steps in [Multiple custom function source files](#multiple-custom-function-source-files) manually.
+
+1. Open a Command Prompt or bash shell and, in the root of the project, run `npm install custom-functions-metadata-plugin`.
+1. Open the webpack.config.js file and add the following line at the top: `const CustomFunctionsMetadataPlugin = require("custom-functions-metadata-plugin");`.
+1. Scroll down to the `plugins` array and add the following to the top of the array. Change the `input` path and filename as needed to match your project, but the `output` value must be "functions.json". If you're using TypeScript, use the \*.ts source file name, *not* the transpiled \*.js file.
+
+   ```js
+   new CustomFunctionsMetadataPlugin({
+      output: "functions.json",
+      input: "./src/functions/functions.js", 
+   }),
+   ```
+
+### Multiple custom function source files
+
+If, and only if, you have organized your custom functions into multiple source files, there are additional steps. 
+
+1. In the webpack.config.js file, replace the string value of `input` with an array of string URLs that point to each of the files. The following is an example:
+
+   ```js
+   new CustomFunctionsMetadataPlugin({
+      output: "functions.json",
+      input: [
+               "./src/functions/someFunctions.js", 
+               "./src/functions/otherFunctions.js"
+             ], 
+   }),
+   ```
+
+1. Scroll to the `entry.functions` property, and replace its value with the same array you used in the preceding step. The following is an example:
+
+   ```js
+   entry: {
+      polyfill: ["core-js/stable", "regenerator-runtime/runtime"],
+      taskpane: ["./src/taskpane/taskpane.js", "./src/taskpane/taskpane.html"],
+      functions: [
+               "./src/functions/someFunctions.js", 
+               "./src/functions/otherFunctions.js"
+             ],
+    },
+   ```
+
+## Run the tool
+
+You don't have to do anything to run the tool. When webpack runs, it creates the functions.json file and puts it in memory in development mode, or in the /dist folder in production mode.
```

Comment:
Making "Webpack" capitalization consistent with rest of article (as a proper noun) 
```suggestion
You don't have to do anything to run the tool. When Webpack runs, it creates the functions.json file and puts it in memory in development mode, or in the /dist folder in production mode.
```